lte/gateway/python/integ_tests/defs.mk

```diff
@@ -251,6 +251,8 @@ s1aptests/test_s1_handover_cancel.py \
 s1aptests/test_s1_handover_failure.py \
 s1aptests/test_s1_handover_timer_expiry.py \
 s1aptests/test_attach_and_mme_restart_loop_detach_and_mme_restart_loop_multi_ue.py \
+s1aptests/test_attach_detach_with_non_nat_dhcp_multi_ue.py \
```

Comment:
What is the reasoning for making them non-sanity tests? Could they be part of the sanity tests or is the setup wrong?

Reply:
I would have the same question.

Reply:
The setup doesn't matter, it has its own setup steps within the test and it doesn't need anything else. We could put the in the sanity tests if you think it is more suitable, I don't have any strong opinion on where to put them.

Reply:
Well, sanity tests are checked in the CI workflows, while the non-sanity tests are just lying around in the code base. Therefore I would prefer to move them out of the nonsanity checks to be tested regularly.